RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Row/NonFungibleAssetRow+View.swift

```diff
@@ -100,7 +100,7 @@ extension NonFungibleAssetList.Row.View {
 
 				HStack {
 					NFTIDView(
-						id: asset.id.nonFungibleLocalId.toUserFacingString(),
+						id: asset.id.nonFungibleLocalId.formatted(.default),
```

Comment:
Is this the only place? I believe we should properly abreviate everywhere?

Reply:
In `AddressView` it is already formatted correctly. Do we display `nonFungibleLocalId` in any other places?

Reply:
This is the only display usage of `toUserFacingString()`, so I believe we don't need to update it anywhere else.

So if we have any other incorrect abbreviation, it is because we are using an incorrect `AddrressFormat`, which I don't recall of any case where this is happening.

Reply:
yeah looks good, just tested.